test/qa-tests/jstests/import/types.js

```diff
@@ -74,7 +74,7 @@
   var oid = ObjectId();
   var irregularObjects = {
     _id: ObjectId(),
-    a: DBPointer('namespace', oid),
+    a: DBRef('namespace', oid),
```

Comment:
not sure if this change is correct - DBPointer was being marshalled differently by the driver and our results expect a DBRef.

Reply:
So DBRef isn't actually a bson type; it's more of a convention. DBPointer _is_ defined by the bsonspec and should be round tripped correctly. The comment above saying it will turn into a DBRef is hopefully now wrong with the new driver update.

I'd say keep the DBRef testing in to ensure it still works (it should, a DBRef is just an embedded Document), but also keep DBPointer in there and ensure it also round trips.

Reply:
Agreed.   I think we should expect DBPointer to be handled accurately, even if deprecated.

Reply:
Added a DBPointer in the previous commit before your comments.